modules/admin_manual/pages/configuration/user/oidc/oidc.adoc

```diff
@@ -48,13 +48,49 @@ Setting up ownCloud Server to work with OpenID Connect requires a couple of comp
 +
 * `'http.cookie.samesite' \=> 'None',`
 +
-See `config.sample.php` and {schemeful-samesite-url}[Schemeful Same-Site] for examples and details.
+See xref:configuration/server/config_sample_php_parameters.adoc#define-how-to-relax-same-site-cookie-settings[config.sample.php] and {schemeful-samesite-url}[Schemeful Same-Site] for examples and details.
```

Comment:
I don't know the way references happen. But this one has `.adoc` and the similar one below has `.html` 
Do either work?

Reply:
🤦‍♂️ you are right, shame on me... fixed